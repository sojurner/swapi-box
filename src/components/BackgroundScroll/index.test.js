import React from "react";

import { BackgroundScroll } from "./index";

describe("BackgroundScroll", () => {
  it("should match snapshot with data object passed", () => {
    let mockFilm = {
      title: "The Empire Strikes Back",
      openingCrawl: "wooooorrrrds",
      episode: 5,
      releaseDate: "1980-05-17"
    };
    const wrapper = shallow(<BackgroundScroll {...mockFilm} />);
    expect(wrapper).toMatchSnapshot();
  });

  it("should match snapshot without data object passed", () => {
    let mockFilm = {};
    const wrapper = shallow(<BackgroundScroll {...mockFilm} />);
    expect(wrapper).toMatchSnapshot();
  });
});
/* global shallow */