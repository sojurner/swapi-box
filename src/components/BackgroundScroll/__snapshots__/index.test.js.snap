// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackgroundScroll should match snapshot with data object passed 1`] = `
<div
  className="Background__Container"
>
  <div
    className="Background__Fade"
  />
  <section
    className="Background__Scroll"
  >
    <div
      className="Background__Crawl"
    >
      <div
        className="Background__Title"
      >
        <p>
          Episode 
          5
        </p>
        <h1>
          The Empire Strikes Back
        </h1>
      </div>
      <p>
        wooooorrrrds
      </p>
      <p>
        1980-05-17
      </p>
    </div>
  </section>
</div>
`;

exports[`BackgroundScroll should match snapshot without data object passed 1`] = `
<div
  className="Background__Container"
>
  <div
    className="Background__Fade"
  />
  <section
    className="Background__Scroll"
  >
    <div
      className="Background__Crawl"
    >
      <div
        className="Background__Title"
      >
        <p>
          Episode 
        </p>
        <h1 />
      </div>
      <p />
      <p />
    </div>
  </section>
</div>
`;
